src/state/process.rs

```diff
@@ -203,6 +203,35 @@ where
         }
     }
 
+    fn handle_resp(&mut self, msg: Option<VerifyResp<S>>) -> ConsensusResult<()> {
+        let resp = msg.ok_or_else(|| ConsensusError::Other("Event sender dropped".to_string()))?;
+        if resp.epoch_id != self.epoch_id {
+            return Ok(());
+        }
+
+        let epoch_hash = resp.epoch_hash.clone();
+        self.is_full_transcation
+            .insert(epoch_hash, resp.full_txs.is_some());
+
+        if let Ok(qc) = self
```

Comment:
should first check precommitQC, if exist, trigger commit, else check prevoteQC.

Reply:
and should search qc by epoch_hash, not by self.epoch_id and self.round.

Reply:
In fact, the process of check prevoteQC is a little complex. You should first search qc by epoch_hash, and then compare the output qcs' max round, if qcs' max round >= self.round, trigger PrevoteQC.